src/Maestro/Microsoft.DotNet.Maestro.Tasks/src/PushMetadataToBuildAssetRegistry.cs

```diff
@@ -100,7 +100,13 @@ private List<BuildData> GetBuildManifestsMetadata(string manifestsFolderPath)
 
                     foreach (Package package in manifest.Packages)
                     {
-                        AddAsset(assets, package.Id, package.Version, manifest.Location, "NugetFeed", package.NonShipping);
+                        AddAsset(
+                            assets, 
+                            package.Id, 
+                            package.Version, 
+                            manifest.InitialAssetsLocation ?? manifest.Location,
```

Comment:
>manifest.InitialAssetsLocation ?? manifest.Location [](start = 28, length = 51)

What is the difference between InitialAssetsLocation and Location and what's the relation of each with "NugetFeed" and "Container"?

Reply:
I plan to remove `Location` eventually and use only `InitialAssetsLocation` because I think the original name is not precise enough. The original "location" of the manifests where a nuget feed, that's why the location type was "NugetFeed". 

We are changing the initial location of the assets to be Pipeline Artifacts, which is a kind of "Container". Maybe we should add a new location type? @alexperovich @mmitche